apps/logo-quiz/src/store/levels/types.ts

```diff
@@ -1,12 +1,12 @@
-import { Level } from '@logo-quiz/models';
-import { AxiosError } from 'axios';
+import { Level } from "@logo-quiz/models";
```

Comment:
why using double quotes?

Reply:
Sorry I had prettier enabled with the wrong config. Will fix it.